frontend/test/pytest/test_gradient.py

```diff
@@ -1200,6 +1200,64 @@ def circuit(phi, psi):
     assert len(result[1]) == 2
 
 
+def test_calssical_kwargs():
+    """Test the gradient on a classical function with keyword arguments"""
+
+    @qjit
+    def f1(x, y, z):
+        return x * (y - z)
+
+    result = qjit(grad(f1, argnum=0))(3.0, y=1.0, z=2.0)
```

Comment:
Do you what happens if argnum refers to a non positional argument? Not sure what happens in Jax neither